### ISSUE TITLE
Restarting a dall-e or vqvae run

### ISSUE THREAD

**Issue by TheodoreGalanos** [lucidrains/DALLE-pytorch#85]
Wonder what is the best way to restart training a vqvae or dall-e models. I'm mostly talking about learning rate, temperature schedules, etc. Would manually adjusting those be the best way right now?

Or would wandb be able to do that for me perhaps?

Thanks!

**Comment by TheodoreGalanos** [comment]
Nevermind, quite straight forward to load model['weights'] into the model! I'm just manually changing learning rate and temperature, but that's okay.
